tests/MCNPinput_test.py

```
@@ -917,3 +917,21 @@ def test_add_SDDR_dose_function(self):
             "DF14 0.0485 0.1254 0.2050 0.2999 0.3381 0.3572"
             in inp.other_data["DF14"].lines[0]
         )
+
+    def test_column_format(self, tmp_path):
+        with as_file(resources_inp.joinpath("column_format.i")) as FILE1:
+            inp = Input.from_input(FILE1)
+
+        outfile = tmp_path / "column_format_output.i"
+        inp.write(outfile)
+
+        inp2 = Input.from_input(outfile)
```

Ruff (0.15.2)

[error] Local variable `inp2` is assigned to but never used

Remove assignment to unused variable `inp2`

(F841)